docs/content/dagster-cloud/getting-started.mdx

```diff
@@ -358,19 +359,45 @@ When finished, [continue to the next step](#step-4-set-up-environment-variables-
 </TabItem>
 <TabItem name="Other CI/CD provider">
 
-For continuous integration using a CI/CD provider other than GitHub, your system should:
+For continuous integration using a CI/CD provider other than GitHub, your system should use the `dagster-cloud ci` subcommand to deploy code locations to Dagster Cloud.
 
-1. Build a Docker image containing your Dagster code.
-2. Push the image to a registry your agent can access.
-3. Deploy the update to Dagster Cloud using the `dagster-cloud` CLI. For example:
+Ensure that you have created a `dagster_cloud.yaml` file as described in [Syncing the workspace](/dagster-cloud/managing-deployments/code-locations#managing-code-locations-using-the-dagster-cloud-cli#syncing-the-workspace). The following steps are typically implemented in the CI/CD workflow:
 
-   ```shell
-   dagster-cloud workspace add-location YOUR_LOCATION_NAME \
-       --image LOCATION_IMAGE \
-       --package-name PACKAGE_NAME \
-   ```
+1. Set the build environment variables:
+
+   - `DAGSTER_CLOUD_ORGANIZATION` - the name of your organization in Dagster Cloud.
+   - `DAGSTER_CLOUD_API_TOKEN` - a token created in Dagster Cloud.
+   - `DAGSTER_BUILD_STATEDIR` - path to a blank or nonexistant temporary directory on the build machine. This directory is used to store local state during the build.
+
+2. Run the configuration check:
+
+       dagster-cloud ci check --project-dir=.
```

Comment:
````suggestion
2. Run the configuration check:

   ```shell
   dagster-cloud ci check --project-dir=.
   ```
````

Make sure to put code snippets into fenced blocks and include a language. CSS isn't applied correctly without it